fs/bcachefs/chardev.c

```diff
@@ -523,7 +523,7 @@ static long bch2_ioctl_read_super(struct bch_fs *c,
 	ret = copy_to_user((void __user *)(unsigned long)arg.sb,
 			   sb, vstruct_bytes(sb));
 err:
-	if (ca)
+	if (!IS_ERR(ca))
```

Comment:
Can you change that to if (!IS_ERR_OR_NUL(ca)) ?

Reply:
Good point. Updated and retested.